### PR TITLE
OpsCenter 6.8.32 Release

### PR DESCRIPTION
**Pull request by vimal-ds**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

File: OpsCenter_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.32
+11 Jan 2024
+
+## Backup Service
+* Add the missing key to details-sstables so opsc won't raise an incorrect error as backup failure (OPSC-17423)
+* Improved test cases to add more value to fetch event store. (OPSC-17419)
+* Clean stale backup/snapshot data when previous cleanup or the job failed in Sync Retry. (OPSC-17186)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:9]:
```suggestion
* Clean-up stale backup/snapshot data when previous cleanup job failed or the job failed during Sync Retry step. (OPSC-17186)
```

File: OpsCenter_6.8_Release_Notes.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed bugs when running a backup using {{allow_one_failure}} when one agent is down. (OPSC-17421)
+
+## Backup Service,UI
+* Added Commitlog backups cache refresh button in commitlogs dialog. (OPSC-17403)
+
+## Core
+* Publish failover and missing heartbeat after heartbeat_reply_period through event services (OPSC-17212)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:16]:
```suggestion
* Publish failover and missing heartbeat after `heartbeat_reply_period` through event services (OPSC-17212)
```

File: OpsCenter_6.8_Release_Notes.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+
+## Core,TestEng
+* Scripts for load testing stomp service (OPSC-17443)
+
+## UI
+* Limit session to User Agent and IP (OPSC-17278)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:22]: limit to how much?

**Review by msmygit — CHANGES_REQUESTED** [review]

File: OpsCenter_6.8_Release_Notes.md (hunk 3 above)
Comment (vimal-ds) [reply OpsCenter_6.8_Release_Notes.md:22]: single session

File: OpsCenter_6.8_Release_Notes.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.32
+11 Jan 2024
+
+## Backup Service
+* Add the missing key to details-sstables so opsc won't raise an incorrect error as backup failure (OPSC-17423)
+* Improved test cases to add more value to fetch event store. (OPSC-17419)
+* Clean-up stale backup/snapshot data when previous cleanup job failed or the job failed during Sync Retry step. (OPSC-17186)
+* Fixed bugs when running a backup using {{allow_one_failure}} when one agent is down. (OPSC-17421)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:10]:
```suggestion
* Fixed bugs when running a backup using `{{allow_one_failure}}` when one agent is down. (OPSC-17421)
```

**Review by msmygit — CHANGES_REQUESTED** [review]

File: OpsCenter_6.8_Release_Notes.md (hunk 4 above)
Comment (msmygit) [reply OpsCenter_6.8_Release_Notes.md]: I know it's too late in the game, but shouldn't we name that field as `{{allow_one_agent_failure}}` instead for clarity?

Comment (msmygit) [reply OpsCenter_6.8_Release_Notes.md]: @orion104 thoughts?

**Review by msmygit — APPROVED** [review]
Release notes LGTM, but I still have question on the actual implementation variable name used. See my earlier comment

File: OpsCenter_6.8_Release_Notes.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed bugs when running a backup using `{{allow_one_failure}}` when one agent is down. (OPSC-17421)
+
+## Backup Service,UI
+* Added Commitlog backups cache refresh button in commitlogs dialog. (OPSC-17403)
+
+## Core
+* Publish failover and missing heartbeat after `heartbeat_reply_period` through event services (OPSC-17212)
+
+## Core,TestEng
+* Scripts for load testing stomp service (OPSC-17443)
```
Comment (orion104) [inline OpsCenter_6.8_Release_Notes.md:19]: Remove completely. We only need release notes for code changes that are customer facing. Tests and internal tool don't need release notes.

Comment (orion104) [inline OpsCenter_6.8_Release_Notes.md:19]: Remove completely. We only need release notes for code changes that are customer facing. Tests and internal tool don't need release notes.

**Review by orion104 — APPROVED** [review]

**Pull request merged**